src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,132 @@
+import classNames from 'classnames';
+import { ChangeEvent } from 'react';
+import { Link, useSearchParams } from 'react-router-dom';
+import { getSearchWith } from '../../utils/searchHelper';
+import { SearchLink } from '../SearchLink';
+
+export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const currentQuery = searchParams.get('query') || '';
+  const currentSex = searchParams.get('sex');
+  const centuries = searchParams.getAll('centuries') || [];
```

Comment:
```suggestion
  const centuries = searchParams.getAll('centuries');
```
if `centuries` is `null` `searchParams.getAll` returns an empty array by default